python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -9,6 +9,7 @@
 from dask.utils_test import hlg_layer
 
 import cudf
+from cudf.testing._utils import expect_warning_if
 
 import dask_cudf
 from dask_cudf.groupby import OPTIMIZED_AGGS, _aggs_optimized
```

Comment:
probably makes sense to update `OPTIMIZED_AGGS` and [`_redirect_aggs`](https://github.com/rapidsai/cudf/blob/2c70971ecc66960dcf4bfb2fc6618c7f9f60980f/python/dask_cudf/dask_cudf/groupby.py#L664C5-L664C19) to use `list`.

Reply:
@rjzamora Done 👍